oximeter/oximeter/schema/http-service.toml

```diff
@@ -14,7 +14,7 @@ description = "Duration for the server to handle a request"
 units = "seconds"
 datum_type = "histogram_f64"
 versions = [
-    { added_in = 1, fields = [ "route", "method", "status_code" ] }
+    { added_in = 1, fields = [ "operation_id", "status_code" ] }
```

Comment:
I'm still partway through supporting actual updates to these schema. Until then, we're getting away with this because we drop the data from the old schema. When I _do_ finish that work, then we'd need to add a new `versions` entry, describing the new fields.